### PR TITLE
SUBMARINE-654. [WEB] Set default parameter when creating experiment

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?
Set placeholder in experiment name, description, command, make the first-time user more easily to use this UI.

### What type of PR is it?
[Improvemen]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-654

### How should this be tested?
https://travis-ci.org/github/pingsutw/hadoop-submarine/builds/733020291

### Screenshots (if appropriate)
Screenshot from 2020-10-05 23-53-02

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-workbench/workbench-web-ng/src/app/interfaces/experiment-spec.ts [hunk 1]
```diff
@@ -53,5 +53,5 @@ export interface ExperimentSpec {
   meta: ExperimentMeta;
   environment: EnvironmentSpec;
   spec: Specs;
-  code: Code;
+  code?: CodeSpec;
```
Comment (xunliu) [inline submarine-workbench/workbench-web-ng/src/app/interfaces/experiment-spec.ts:56]: `code?` Is there a problem here?

Comment (pingsutw) [reply submarine-workbench/workbench-web-ng/src/app/interfaces/experiment-spec.ts:56]: "?" hear mean optional, so we could set input null if we don't want to sync up code from Github.

Comment (xunliu) [reply submarine-workbench/workbench-web-ng/src/app/interfaces/experiment-spec.ts:56]: OK.

**Review by xunliu — APPROVED** [review]
LGTM

**Pull request closed without merging**